giskard-frontend/src/views/main/project/PredictionResults.vue

```diff
@@ -184,35 +214,70 @@ export default class PredictionResults extends Vue {
     else return undefined
   }
 
+  // Breakpoints for number of displayed characters on the chart
+  get maxCharsCategory(){
+    switch (this.$vuetify.breakpoint.name) {
```

Comment:
The problem is that the ref attribute is available only when the html element is mounted. However we want to modify directly the options of the charts (by modifying the maxcharscategory). That's seems to be impossible

Reply:
I see what you mean, in this case you can get a reference of card element that contains the results, then you can get its width with `resultcard.$el.clientWidth`

Reply:
I've done that for the Result card.
However for the popover, I struggled for an hour to get the element by this way. I red lots of forums and it seems to be fuzzy. To resolve that, I assumed that the popover is 90% of the window width. In every case this will be true so it works perfectly and I can't see a scenario where it's buggy.